libafl_qemu/src/command/parser.rs

```diff
@@ -138,7 +138,7 @@ where
         qemu: Qemu,
         arch_regs_map: &'static EnumMap<ExitArgs, Regs>,
     ) -> Result<Self::OutputCommand, CommandError> {
-        let input_virt_addr: GuestVirtAddr = qemu.read_reg(arch_regs_map[ExitArgs::Arg1])?;
+        let input_virt_addr: GuestVirtAddr = qemu.read_reg(arch_regs_map[ExitArgs::Arg1])?.into();
```

Comment:
@rmalmain Don't really understand why GuestVirtAddr can be a different size to GuestReg, but anyway not a big issue

Reply:
this is more for semantic purposes, most of the time `len(GuestReg) = len(GuestVirtAddr) >= len(GuestPhysAddr)`.
so we could just use GuestReg everywhere in theory i think.
it's just convenient to know what exactly we manipulate with the type.